tests/performance/local_replica.xml

```diff
@@ -0,0 +1,3 @@
+<test>
+    <query>select sum(number) from remote('127.0.0.{{1|2}}', numbers_mt(1000000000)) group by bitAnd(number, 1)</query>
```

Comment:
Hm, not sure that this query covers that case, since this reads lots of rows and also it uses numbers_mt

Reply:
Right in this case the number of threads was limited to 1.